app/process/handler/crypto-transfer/crypto-transfer.go

```diff
@@ -126,7 +129,7 @@ func (cth *CryptoTransferHandler) Handle(payload []byte) {
 		}
 	}
 
-	validFee, err := fees.ValidateExecutionFee(ctm.Fee)
+	validFee, err := cth.feeCalculator.ValidateExecutionFee(ctm.Fee, ctm.Amount, ctm.GasPriceGwei)
```

Comment:
With the addition of this line, the status of the current processed tx will never be updated to f.e. `StatusInsufficientFee`